src/modules/Todo.js

```diff
@@ -0,0 +1,68 @@
+const descriptionInput = document.getElementById('description-input');
+let allToDos = [];
+const todosList = document.getElementById('todos');
+
+const deleteToDoItem = (index) => {
+  allToDos.splice(index, 1);
+  todosList.innerHTML = '';
+  localStorage.setItem('alltasks', JSON.stringify(allToDos));
+};
```

Comment:
#### Good work displaying the tasks but there are some aspects you can improve
- [ ] As stated in the project requirement, Kindly ensure that - By default new tasks should have the property `completed` set to `false` and the property `index` set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5).

![image](https://user-images.githubusercontent.com/98453979/228654460-11a98234-d261-4a2c-87f9-c85f94a15d2a.png)

- I tried clicking on the `three doted icon` on your design and I noted the tasks were being deleted but that could only be displayed after I refresh the page. Kindly try following [this](https://www.youtube.com/watch?v=AcUd-_Yjjqg) video to guide you on how to implement the delete functionality. Also note that deleting a task should update all remaining items' indexes, so they represent the current list order and are unique(i.e. if you're deleting the first task index 1 from the list, the index of the next task(2) should set to 1).
- I could not edit your tasks. Kindly ensure that all your tasks can be edited and change saved in the local storage as stated in the project requirement

![image](https://user-images.githubusercontent.com/98453979/228655134-1c5d0590-e980-4724-bbc0-720f17ac61c5.png)